Businesss/Business.csproj

```diff
@@ -1,12 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
-
   <ItemGroup>
-    <ProjectReference Include="..\Entities\Entities.csproj" />
-    <ProjectReference Include="..\DataAccess\DataAccess.csproj" />
+    <ProjectReference Include="..\Entities\Entities.csproj"/>
+    <ProjectReference Include="..\DataAccess\DataAccess.csproj"/>
   </ItemGroup>
-
-</Project>
+  <ItemGroup>
+    <PackageReference Include="Autofac" Version="6.3.0"/>
+    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0"/>
```

Comment:
I suggest you to have a look at Directory.build.targets . It is a nice feature to handle all versions under one solution from a single file.

https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-dot-targets-files?view=vs-2022